src/cmd-compress

```diff
@@ -62,6 +52,25 @@ else:
 
 print(f"Targeting build: {build}")
 
+# that's the tool default
+gzip_level = 6
+if args.fast:
+    args.compressor = 'gzip'
+    gzip_level = 1
+elif not args.compressor:
+    workdir = os.getcwd()
+    arches = builds.get_build_arches(build)
+    # image.json isn't arch-dependent, so just extract it from the first arch
+    buildmeta = GenericBuildMeta(workdir=workdir, build=build, basearch=arches[0])
+    extract_image_json(workdir, buildmeta['ostree-commit'])
+    with open(os.path.join(workdir, 'tmp/image.json')) as f:
+        image_json = json.load(f)
```

Comment:
This requires the actual ostree ociarchive to be local, right? If that's the case we might need to lower this into the `compress` path so that `cosa buildfetch --stream=testing-devel --artifact=qemu && cosa decompress` doesn't require the ociarchive to be present.

Reply:
https://github.com/coreos/coreos-assembler/pull/3245